code/datums/wires/airlock.dm

```diff
@@ -170,6 +166,7 @@
 
 		if(WIRE_SHOCK)
 			//one wire for electrifying the door. Sending a pulse through this electrifies the door for 30 seconds.
+			A.shock(usr, 50)
```

Comment:
`usr` isn't acceptable, this proc receives the `user` already as a parameter, adjust the proc signature and use that instead